provider/github/diff_test.go

```diff
@@ -138,6 +138,44 @@ func TestParseHunks(t *testing.T) {
 			{Type: lineContext, Lines: 2},
 		},
 	}}, hunks)
+
+	hunks, err = parseHunks(largeHunk)
```

Comment:
could you please move it to separate test like `TestParseHunks__issue_165`?

it's really hard to understand are those numbers correct or not. With the link to the issue, a developer later can actually generate such diff manually using commit hashes and add debug info wherever necessary if this test is failing.

Reply:
Addressed in 00dc38c